libs/client/cornflow_client/core/application.py

```diff
@@ -181,6 +184,12 @@ def solve(
 
         instance_checks = SuperDict(inst.check())
 
+        inst_errors = inst.validate_checks(instance_checks)
```

Comment:
i wouldn't use "inst_errors" again because it can be confusing with the errors of the instance schema. I think "inst_checks_errors" is better. 
And the same with "inst_errors". I would prefer "inst_squema_errors"

If anything, two different names. If you like to keep "inst_errors" for me that would be for the checks. And for the squema of the instance i would use inst_squema_errors

Reply:
I have changed the way it is done. See if you like it.